DragaliaAPI/Services/Game/SavefileService.cs

```diff
@@ -630,6 +645,7 @@ public IQueryable<DbPlayer> Load()
             .Include(x => x.WeaponSkinList)
             .Include(x => x.WeaponPassiveAbilityList)
             .Include(x => x.EquippedStampList)
+            .Include(x => x.PartyPower)
```

Comment:
Please can you include this in the GraphQL schema as well?

Reply:
will do